### PR TITLE
refactor(dropdown): [dropdown] refactoring styles and variable for dropdown

### PR DESCRIPTION
**Pull request by MomoPoppy**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **New Features**
	- Updated dropdown menus to display new labels, enhancing clarity for users.
	- Expanded selection options in the `tiny-select` component to include "广州" (Guangzhou) and "武汉" (Wuhan).
  
- **Bug Fixes**
	- Removed unnecessary CSS styles that may have affected dropdown button appearance.

- **Style**
	- Introduced new styling conventions and improved organization for dropdown components.
	- Adjusted font sizes and colors for better visual consistency across dropdown elements.
	- Updated variable names for better clarity and consistency in styling.

### CODE REVIEW

File: packages/vue/src/dropdown/src/pc.vue [hunk 1]
```diff
@@ -168,7 +168,7 @@ export default defineComponent({
           round={round}
           disabled={disabled}
           type={type}
-          class={`tiny-dropdown__border ${state.visible ? 'is-expand' : ''}${showIcon ? ' is-show-icon ' : ''}`}
+          class={`tiny-dropdown__border ${state.visible ? 'is-expand' : ''}${showIcon ? ' is-show-icon ' : ''} ${triggerClass}`}
           reset-time={0}>
```
Comment (zzcr) [inline packages/vue/src/dropdown/src/pc.vue:172]: 这里需要评估下会不会对saas主题造成影响？

File: packages/theme/src/tabs/vars.less [hunk 2]
```diff
@@ -39,8 +39,6 @@
   --ti-tabs-dropdown-border-radius: var(--ti-common-border-radius-normal, 2px);
   // 标签页下拉选项文本色(hide)
   --ti-tabs-dropdown-li-text-color: var(--ti-common-color-info-normal, #252b3a);
-  // 标签页内容字号
-  --ti-tabs-content-font-size: var(--ti-common-font-size-2, 16px);
   // card类型(无size属性)标签项高度
```
Comment (chenxi-20) [inline packages/theme/src/tabs/vars.less]: 这个为什么会改到tabs的组件

File: packages/vue/src/dropdown/src/pc.vue (hunk 1 above)
Comment (MomoPoppy) [reply packages/vue/src/dropdown/src/pc.vue:172]: image
image
image
已验证

Comment (MomoPoppy) [reply packages/vue/src/dropdown/src/pc.vue:172]: image
这里的改动一开始导致图标没对齐，定位后发现应该是原有组件样式问题，然后在demo里设置间距对齐居中。
修改后不需要demo的间距，组件图标是对齐居中的。

File: packages/theme/src/tabs/vars.less (hunk 2 above)
Comment (MomoPoppy) [reply packages/theme/src/tabs/vars.less]: 原计划全局统一设置默认字号14px，这样组件就不需要重复写字号样式。但tab组件设置内容区字号16px，就会导致demo里的默认字号是16px。规范里也没有规定内容区的默认字号是16px，这是个错误样式
现计划不全局设置字号，就暂不影响demo里的组件
image

**Review by zzcr — APPROVED** [review]

**Pull request merged**